packages/dx-react-bootstrap4/components/popover.jsx

```diff
@@ -127,4 +137,5 @@ Popover.defaultProps = {
   isOpen: false,
   placement: 'auto',
   toggle: undefined,
+  arrowComponent: undefined,
```

Comment:
Why the default prop for `arrowComponent` is `undefined`?

Maybe we should use a function `() => null`?

Or the next syntax for default prop `arrowComponent: DefaultArrowomponent`